### PR TITLE
enhancements/update/update-blocker-lifecycle: Pivot to "update recommendations"

### PR DESCRIPTION
**Pull request by wking**
Users are occasionally confused by the "blocked edges" wording, which is internal shorthand.

The fact that update recommendations are tracked as a directed graph (hence "edges") isn't all that relevant for most users, so talk about "updates" instead of "edges".

While we occasionally drop update recommendations to reduce the chances that users update into serious issues, from [docs][1]:

> When an update recommendation is supported, it remains supported for the life of 4.10, even if the update recommendation is later dropped or made conditional.

Updating is still possible, we just don't think it's a good idea (unless maybe you're elbow deep in a support case, and a support tech decides to recommend an update based on context that is not available to the update service).  So replace "blocked" with "not recommended", to convey the "still possible" aspect.

Also ask for some more specifics in the impact statement template, as we gear up to support conditional updates fff9d3c349 (#821).  I haven't gone as far as "tell us what PromQL to use in our conditional recommendation", but I'm easing things in that direction.

[1]: https://docs.openshift.com/container-platform/4.10/updating/understanding-upgrade-channels-release.html#upgrade-version-paths_understanding-upgrade-channels-releases

### CODE REVIEW

File: enhancements/update/update-blocker-lifecycle/README.md [hunk 1]
```diff
@@ -5,17 +5,19 @@ authors:
...
+last-updated: 2022-04-19
 status: implementable
 ---
 
 # Update-blocker Lifecycle
 
 We occasionally have bugs which impact update success or the stability of the target release.
-When that happens, we protect users by [removing the impacted updates from our update recommendations][graph-data-block].
-This enhancement describes the process used to identify these bugs and determine which edges should be blocked because of them.
+When that happens, we protect users by [removing or qualifying the impacted updates from our update recommendations][graph-data-block].
```
Comment (LalatenduMohanty) [inline enhancements/update/update-blocker-lifecycle/README.md:19]: I do not think `we protect users by qualifying the impacted updates from our update recommendations` communicates the same thing you want to communicate

File: enhancements/update/update-blocker-lifecycle/README.md [hunk 2]
```diff
@@ -69,35 +71,35 @@ Write tooling that automatically:
...
+We're asking the following questions to evaluate whether or not this bug warrants changing update recommendations from either the previous X.Y or X.Y.Z.
 The ultimate goal is to avoid delivering an update which introduces new risk or reduces cluster functionality in any way.
 Sample answers are provided to give more context and the ImpactStatementRequested label has been added to this bug.
 When responding, please remove ImpactStatementRequested and set the ImpactStatementProposed label.
 The expectation is that the assignee answers these questions.
 
-Who is impacted?  If we have to block upgrade edges based on this issue, which edges would need blocking?
-* example: Customers upgrading from 4.y.Z to 4.y+1.z running on GCP with thousands of namespaces, approximately 5% of the subscribed fleet
-* example: All customers upgrading from 4.y.z to 4.y+1.z fail approximately 10% of the time
+Who is impacted?  Which 4.y.z to 4.y'.z' updates?  Which types of clusters?  How does a cluster admin determine they are vulnerable?
```
Comment (LalatenduMohanty) [inline enhancements/update/update-blocker-lifecycle/README.md:80]: `How does a cluster admin determine they are vulnerable?` should be in a separate line.

File: enhancements/update/update-blocker-lifecycle/README.md [hunk 3]
```diff
@@ -69,35 +71,35 @@ Write tooling that automatically:
...
+* example: Customers upgrading from 4.y.Z to 4.y+1.z running on GCP with thousands of namespaces, approximately 5% of the subscribed fleet.  Check your vulnerability with `oc ...` or the following PromQL `count (...) > 0`.
+* example: All customers upgrading from 4.y.z to 4.y+1.z fail approximately 10% of the time.  Check your vulnerability with `oc adm upgrade` to show your current cluster version.
 
-What is the impact?  Is it serious enough to warrant blocking edges?
-* example: Up to 2 minute disruption in edge routing
-* example: Up to 90 seconds of API downtime
-* example: etcd loses quorum and you have to restore from backup
+What is the impact?  Is it serious enough to warrant removing update recommendations?  How does a cluster admin determine they are getting bitten?
```
Comment (LalatenduMohanty) [inline enhancements/update/update-blocker-lifecycle/README.md:84]: `How does a cluster admin determine they are vulnerable?` and `How does a cluster admin determine they are getting bitten?` is almost the same thing IMO.

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

File: enhancements/update/update-blocker-lifecycle/README.md (hunk 3 above)
Comment (wking) [reply enhancements/update/update-blocker-lifecycle/README.md:84]: the distinction is that "vulnerable" would go into conditional update preconditions.  E.g. "if I have `$CONFIG`, and I update to `$VERSION`, I'll be at risk".  Like [high control plane memory and 4.9.<28 makes you vulnerable to etcd data inconsistency][1].  "Getting bitten" is "the bad thing is currently happening to you" and goes into the recovery KCS.  Like "[`etcdctl endpoint status -w fields --cluster` shows persistent, divergent revisions][1]".  We can protect folks from vulnerabilities without waiting for them to get bitten.  Some folks who we fail to protect from vulnerabilities will _still_ not get bitten.

[1]: https://access.redhat.com/solutions/6849521

File: enhancements/update/update-blocker-lifecycle/README.md (hunk 1 above)
Comment (wking) [reply enhancements/update/update-blocker-lifecycle/README.md]: Reworded to:

```markdown
When that happens, we protect users by [removing update recommendations or qualifying recommendations them with conditional risks][graph-data-block].
```

with af8037bf -> 92525969.

File: enhancements/update/update-blocker-lifecycle/README.md (hunk 3 above)
Comment (wking) [reply enhancements/update/update-blocker-lifecycle/README.md]: Yup.  I've pushed 92525969 -> 18de1352 making all of this more explicit.

File: enhancements/update/update-blocker-lifecycle/README.md (hunk 2 above)
Comment (wking) [reply enhancements/update/update-blocker-lifecycle/README.md]: Line is gone with 92525969 -> 18de1352; please open a new thread if you have line-wrapping issues with the new content :)

**Comment by sdodson** [comment]
/lgtm
/override ci/prow/markdownlint
This amends an existing enhancement

**Pull request merged**